debezium-quarkus-outbox/runtime/src/main/java/io/debezium/outbox/quarkus/internal/AdditionalJaxbMappingProducerImpl.java

```diff
@@ -66,8 +78,20 @@
                 }
             }
         }
-        catch (Exception e) {
-            throw new RuntimeException("Failed to submit OutboxEvent.hbm.xml mapping to Hibernate ORM", e);
+        catch (IOException e) {
+            throw new MappingException("Failed to read OutboxEvent.hbm.xml", e);
```

Comment:
Can we use `DebeziumException` instead? Feels a bit odd to raise a Hibernate exception type here.